### PR TITLE
Add kmermaid Makefile for HIV infected cells to test K2G null hypothesis

### PR DESCRIPTION
**Pull request by olgabot**
Analyzes data downloaded from https://github.com/czbiohub/kh-workflows/pull/14

- Add czbiohub_local profile to not take ALL the cpus"
- Add golumbeanu2018 kmermaid Makefile

### CODE REVIEW

File: kmermaid/golumbeanu2018/Makefile~ [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		--extract_coding_peptide_ksize 15 \
+		-resume \
+		-with-tower \
+		--track_abundance
+
+
+no_k2g_uniprot:
+	nextflow run \
+		${COMMON_FLAGS} \
+		--outdir ${OUTDIR_BASE}/$@ \
```
Comment (pranathivemuri) [inline kmermaid/golumbeanu2018/Makefile~]: what does $@ do?

File: kmermaid/golumbeanu2018/Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+FASTA_DIR=${BASE}rawdata
+WORK_DIR=/home/olga/data_sm/nextflow-intermediates-lrrr
+
+COMMON_FLAGS=nf-core/kmermaid \
+		-r olgabot/fastp \
+		-latest \
+		-profile docker,czbiohub_local \
+		--molecules dna,protein,dayhoff,hp \
+		--ksizes '21,27,33,39,51,63,93' \
+		--log2_sketch_sizes 14 \
```
Comment (pranathivemuri) [inline kmermaid/golumbeanu2018/Makefile]: should this be changed as well?

File: kmermaid/golumbeanu2018/Makefile~ (hunk 1 above)
Comment (olgabot) [reply kmermaid/golumbeanu2018/Makefile~]: It's this magical Makefile thing that inserts the name of the rule! so `$@` is a variable to the rule name

File: kmermaid/golumbeanu2018/Makefile (hunk 2 above)
Comment (olgabot) [reply kmermaid/golumbeanu2018/Makefile]: I'd like to see this, actually. I'm very curious whether we can spot the difference in the protein k-mers
